src/components/paragraphs/sum-course-filter/course-filtering-form.tsx

```diff
@@ -246,7 +246,7 @@ const RefinementInput = ({
         {refinementOptions.map(refinementOption => (
           <label key={refinementOption.value} className="mb-5 flex items-center gap-5 last:mb-0">
             <input
-              className="block h-[2.4rem] w-[2.4rem] cursor-pointer rounded-full border-2 border-black-50 outline-none transition-all checked:border-lagunita-light checked:bg-[length:0px_0px] checked:text-lagunita checked:ring-4 checked:ring-inset checked:ring-white checked:hover:border-lagunita focus:border-lagunita focus:bg-transparent focus:outline-none focus:ring-0 focus:ring-offset-0 checked:focus:border-lagunita checked:focus:bg-lagunita-dark checked:focus:text-lagunita-dark checked:focus:ring-4 checked:focus:ring-inset checked:focus:ring-[#99D7E1] focus-visible:outline-none group-hover:border-lagunita group-hover:bg-transparent checked:group-hover:bg-lagunita-dark checked:group-hover:text-lagunita-dark group-hover:checked:ring-[#99D7E1]"
+              className="block h-[2.4rem] w-[2.4rem] cursor-pointer rounded-full border-2 border-black-50 outline-none transition-all checked:border-lagunita-light checked:bg-[length:0px_0px] checked:text-lagunita checked:ring-4 checked:ring-inset checked:ring-white hocus:border-lagunita hocus:bg-[#99D7E1] hocus:bg-transparent hocus:outline-none hocus:ring-0 hocus:ring-offset-0 checked:hocus:border-lagunita checked:hocus:bg-lagunita-dark checked:hocus:text-lagunita-dark checked:hocus:ring-4 checked:hocus:ring-inset checked:hocus:ring-[#99D7E1] hocus-visible:outline-none group-hocus:border-lagunita group-hocus:bg-transparent checked:group-hocus:bg-lagunita-dark group-hocus:checked:bg-[#99D7E1] checked:group-hocus:text-lagunita-dark"
```

Comment:
I don't see what the `group` is for. There's no group parent, and `checked:group-hocus` on the `<input>` doesn't make much sense to me. I don't think any of these changes are working at this time.

Reply:
I went in to clean up remove the `group-hocus` (along with some `checked:hocus`) that weren't doing anything. I also moved the accessibility fixes from #56 to this PR instead 😃 It's ready for another looksie when you have the chance! Thank you, Mike!